doc/source/Platforms.rst

```diff
@@ -443,11 +458,25 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    vi envs/jedi-ufs.mymacos/common/*.yaml
    vi envs/jedi-ufs.mymacos/site/*.yaml
 
+.. note::
+  When editing the :code:`envs/jedi-ufs.mymacos/spack.yaml` make sure that under the :code:`specs:` section you enter :code:`openmpi` and at least one of the jedi specs such as :code:`jedi-ufs-env`.
+  Here is an example:
+  
+  .. code-block:: yaml
+  
+    specs:
+        - jedi-ufs-env
+        - openmpi
+  
+  This will ensure that you build a complete stack for building the corresponding JEDI bundles.
```

Comment:
See above, this repository is not about JEDI in my opinion. Also, the instructions to start with an empty template (or choose another template) and then add specs with `spack add` do the same, but in a `spack` way.

Reply:
I removed this note.